website/documentation/content/user_documentation.py

```diff
@@ -105,6 +105,75 @@ def querying():
             ''')
 
 
+doc.text('User Interaction', '''
+    `user.find(...)` returns an `UserInteraction` object which provides methods to type text,
```

Comment:
```suggestion
    `user.find(...)` returns a `UserInteraction` object which provides methods to type text,
```

Weird quirk of English: the letter "u" at the beginning of a word _sounds like_ a consonant.